resources/ctia-default.properties

```diff
@@ -12,25 +12,27 @@ ctia.http.show.protocol=http
 ctia.nrepl.port=3001
 ctia.nrepl.enabled=true
 
-ctia.store.actor=es
-ctia.store.feedback=es
-ctia.store.campaign=es
-ctia.store.coa=es
-ctia.store.exploit-target=es
-ctia.store.identity=es
-ctia.store.incident=es
-ctia.store.indicator=es
-ctia.store.judgement=es
-ctia.store.sighting=es
-ctia.store.ttp=es
+# Use ES for all Stores
+ctia.store.default.type=es
+ctia.store.default.uri=http://localhost:9200
+ctia.store.default.indexname=ctia
+ctia.store.actor.indexname=ctia_actor
+ctia.store.campaign.indexname=ctia_campaign
+ctia.store.coa.indexname=ctia_coa
+ctia.store.exploit-target.indexname=ctia_exploit-target
+ctia.store.feedback.indexname=ctia_feedback
+ctia.store.identity.indexname=ctia_entity
+ctia.store.incident.indexname=ctia_incident
+ctia.store.indicator.indexname=ctia_indicator
+ctia.store.judgement.indexname=ctia_judgement
+ctia.store.sighting.indexname=ctia_sighting
+ctia.store.ttp.indexname=ctia_ttp
```

Comment:
Can we split the notion of specifying the store type from setting the store configuration?  I don't really like the _ctia.store.default.type_ property; I prefer the old way.  Instead you could have something like _ctia.store.es.default.uri_, and override that with _ctia.store.es.actor.uri_.  That way it is easy to immediately see what store type is being used without having to scan for a default and then scan again for any overrides.